rendered/manifests/logs-only/configmap-agent.yaml

```diff
@@ -168,6 +168,8 @@ data:
           receivers:
           - fluentforward
           - otlp
+        logs/host:
+          receivers: null
```

Comment:
Please always look at rendered files, this clearly means that something is broken